docs/get-started/getting-started-kubernetes.md

```diff
@@ -0,0 +1,254 @@
+---
+layout: page
+title: Kubernetes
+nav_order: 6
+parent: Getting-Started
+---
+
+# Getting Started with RAPIDS and Kubernetes
+
+This guide will run through how to set up the RAPIDS Accelerator for Apache Spark in a Kubernetes cluster.
+At the end of this guide, the reader will be able to run a sample Apache Spark application that runs
+on NVIDIA GPUs in a Kubernetes cluster.
+
+Kubernetes requires a Docker image to run Spark.  Generally everything needed is in the Docker
+image - Spark, the RAPIDS Accelerator for Spark jars, and the discovery script.  See this
+[Dockerfile.cuda](Dockerfile.cuda) example.
+
+## Prerequisites
+    * Kubernetes cluster is up and running with NVIDIA GPU support
+    * Docker is installed on a client machine
+    * A Docker repository which is accessible by the Kubernetes cluster
+
+These instructions do not cover how to setup a Kubernetes cluster.
+
+Please refer to [Install Kubernetes](https://docs.nvidia.com/datacenter/cloud-native/kubernetes/install-k8s.html) on 
+how to install a Kubernetes cluster with NVIDIA GPU support.
+
+## Docker Image Preparation
+
+On a client machine which has access to the Kubernetes cluster:
+
+1. [Download Apache Spark](https://spark.apache.org/downloads.html).
+   Supported versions of Spark are listed on the [RAPIDS Accelerator download page](../download.md).  Please note that only
+   Scala version 2.12 is currently supported by the accelerator. 
+
+   Note that you can download these into a local directory and untar the Spark `.tar.gz` as a directory named `spark`.
+
+2. Download the [RAPIDS Accelerator for Spark jars](getting-started-on-prem.md#download-the-rapids-jars) and the
+  [GPU discovery script](getting-started-on-prem.md#install-the-gpu-discovery-script).
+  
+   Put the 2 jars -- `rapids-4-spark_<version>.jar`, `cudf-<version>.jar`  and `getGpusResources.sh` in the same directory as `spark`.
```

Comment:
we should clarify this and say in the jars directory within the spark download

Reply:
@tgravescs Actually this means the spark directory + 2 jars + getGpusResources.sh are in the same directory.

Reply:
oh sorry I see.   the docker file in next step will move them.